### PR TITLE
 Change how services are loaded from the env and add support for legacy

### PR DESCRIPTION
**Pull request by eleboucher**
This should fix issue #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,20 +17,26 @@ Then, during container startup, you can use the `waitforservices` command to wai
 
 ## Usage
 
-Without configuration, it finds all TCP services linked to a Docker container via their [environment variables](http://docs.docker.com/userguide/dockerlinks/#environment-variables) and concurrently and repeatedly tries to open a TCP connection to all of them.
+Without configuration, it finds all TCP services specified by the environement variable declared like _\_HOST and _\_PORT (for exemple POSTGRES_HOST and POSTGRES_PORT) and concurrently and repeatedly tries to open a TCP connection to all of them.
```
Comment (tobischo) [inline README.md]: `example`

File: README.md [hunk 2]
```diff
@@ -48,9 +46,19 @@ When all connections are successful, it returns. If one or more services aren't
     When timeout is over and TCP connect or HTTP request were unsucecssful, exit
     with status 1.
 
+## Legacy support
+
+**NOTE**: [Docker link environment variables are deprecated and will be removed](https://docs.docker.com/compose/link-env-deprecated/). We haven't thought about how waitforservices will work then. See [issue 3](https://github.com/Barzahlen/waitforservices/issues/3) for more information. We'd be happy about ideas.
```
Comment (tobischo) [inline README.md]: Technically we have thought about it now: by explicitly setting environment variables following the pattern `<something>_HOST` and `<something>_PORT`

Therefore this actually solves issue #3

**Review by tobischo — CHANGES_REQUESTED** [review]
Looks good, I noticed 1 typo and we probably do not want to say anymore that the `links` going away is not solved.

File: README.md (hunk 2 above)
Comment (eleboucher) [reply README.md]: done in 2012765945c7661c829b1c26403cbcec3093f900

**Pull request merged**